src/WebJobs.Script.WebHost/Program.cs

```diff
@@ -81,12 +81,26 @@ public static IWebHostBuilder CreateWebHostBuilder(string[] args = null)
                 .UseStartup<Startup>();
         }
 
+        internal static void LogVersions(object state)
+        {
+            Console.WriteLine($"HOST_VERSION : {Environment.GetEnvironmentVariable("HOST_VERSION")}");
+            Console.WriteLine($"FUNCTIONS_EXTENSION_VERSION : {Environment.GetEnvironmentVariable("FUNCTIONS_EXTENSION_VERSION")}");
+            Console.WriteLine($"FRAMEWORK : {Environment.GetEnvironmentVariable("FRAMEWORK")}");
+            Console.WriteLine($"FRAMEWORK_VERSION : {Environment.GetEnvironmentVariable("FRAMEWORK_VERSION")}");
```

Comment:
I think you can still get all this data from the last 30 mins result from their log analytics workspace. We don't need to explicitly log environment variables.

Reply:
For sampling, I exported data for last 12 hours for a cluster which had 4 functionapps already running : 
https://microsoft-my.sharepoint.com/:x:/p/shubhamc/EQ1SJNdAXvhCvaKHXMbMSJIBu_4tM_ZEJtWwDBRuUxwkAQ?e=T4DDxU
it does not have the host version. Although out of the 4 apps in the cluster, one of them has. So, I wanted to just publish explicitly. Maybe it is fine to just publish maybe once every 30 mins or so. Would be interested to know your opinion.

Reply:
I don't see any logs in this link above..it's only logging the lock call to the pod. Do we not send the stdout log stream of the pod to customer's Azure Monitor?

Reply:
It is because the lock acquire runs periodically every few seconds/mins and fills up most of the logs. So the log analytics workspace come under Azure monitor only and we can actually see the functionapp pod logs here if lock call logs don't fill it up repeatedly.